lib/kotkowo/plausible/plausible.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Kotkowo.Plausible do
+  @moduledoc false
+  alias Kotkowo.Client.ViewUpdate
+
+  require Logger
+
+  @plausible_key System.fetch_env!("PLAUSIBLE_KEY")
+  @plausible_url System.fetch_env!("PLAUSIBLE_URL")
+  @kotkowo_domain System.fetch_env!("PHX_HOST")
```

Comment:
Wrong. This will set them at compile time.
With the app being built in docker you will have empty strings here always.
Set it in `runtime.exs` and then use: https://hexdocs.pm/elixir/1.18.3/Application.html#get_env/3